### PR TITLE
[torchfuzz] Enhance PointwiseOperator with scalar/broadcast inputs and randomized hyperparameters (#185383)

### PR DESCRIPTION
**Pull request by ddneilson**
Summary:

Backward-compatible enhancements to `PointwiseOperator` and three upstream activation operators:

**`PointwiseOperator` changes (`tensor_pointwise.py`):**

1. `fuzz_inputs_specs` now randomly replaces one of the two inputs with a `ScalarSpec` (30% probability). This exercises the `torch.add(tensor, scalar)` dispatch path that is ubiquitous in real models but was never tested by the fuzzer.

2. `fuzz_inputs_specs` has a secondary 20% probability path that replaces one input's shape with a broadcast-compatible shape (via `_random_broadcast_shape`). This exercises PyTorch's implicit broadcast semantics for binary ops.

3. `symbol` parameter defaults to `""` (was required). This allows subclasses that represent named-function ops (e.g. `torch.maximum`) to skip the symbol without passing a dummy string.

4. Default `torch_op_name` property returns `self.name`. This eliminates the need for every subclass to define the same one-liner property.

5. `AddOperator.codegen` now has a 30% probability of emitting `alpha=<random>` to exercise the `torch.add(a, b, alpha=...)` path. For integer output dtypes, the alpha is truncated to `int`.

**Activation operator changes (`nn_functional.py`):**

6. `GELUOperator.codegen` now randomizes `approximate=` between `"none"` and `"tanh"` (was always `"none"`).

7. `LeakyReLUOperator.codegen` now draws `negative_slope` from `Uniform(0.001, 0.5)` (was hardcoded `0.01`).

8. `ELUOperator.codegen` now draws `alpha` from `Uniform(0.1, 3.0)` (was always the PyTorch default `1.0`).

Test Plan:
These changes were tested as part of a collection of changes to add operator coverage to torchfuzz.

Operator definitions were validated end-to-end using a downstream torchfuzz device plugin for a non-GPU architecture. A test harness ran the fuzzer across all operators in the plugin's template, testing each operator in isolation (--max-depth 1 --supported-ops '<op>') so that failures are attributable to a single operator definition. Per-operator seed counts were derived from coupon-collector analysis: for each operator, the rarest discrete random outcome in fuzz_inputs_specs is identified (probability p_min), then N = ceil(ln(0.05) / ln(1 - p_min)) gives 95% confidence of exercising that path at least once; N is then doubled for margin. This produced tiered counts from 10 seeds (operators with no discrete choices) to 70 seeds (operators with complex mode/dim/dtype interactions). 5830 total invocations across all operator categories.

Failures were automatically classified: "investigate" (fuzzer crashed before generating a program — likely a bug in the operator's fuzz_inputs_specs or codegen) vs "triage" (program was generated but failed at compile or execution time — could be a definition bug or a legitimate backend issue the fuzzer is designed to surface). Three iterative triage-and-fix rounds were completed until all operator-definition bugs were resolved. Remaining triage failures are legitimate backend compilation/execution issues.

Differential Revision: D106395651

### CODE REVIEW

File: tools/experimental/torchfuzz/operators/tensor_pointwise.py [hunk 1]
```diff
@@ -5,21 +5,52 @@
...
-from torchfuzz.tensor_fuzzer import Spec, TensorSpec
+from torchfuzz.tensor_fuzzer import ScalarSpec, Spec, TensorSpec
 from torchfuzz.type_promotion import (
     get_dtype_map,
     get_dtype_name,
     get_promotion_table_for_strings,
 )
 
 
+def _contiguous_stride(size: tuple[int, ...]) -> tuple[int, ...]:
```
Comment (Skylion007) [inline tools/experimental/torchfuzz/operators/tensor_pointwise.py:16]: Use a TypeVarTuple and you can encode that the Tuple is the same length afterwards.

Comment (ddneilson) [reply tools/experimental/torchfuzz/operators/tensor_pointwise.py:16]: @Skylion007  True, but then we can't preserve that the element type is 'int'. I think that part's more important for the type annotation than the length preservation. wdyt?

**Review by bobrenjc93 — APPROVED** [review]
